script.js

```diff
@@ -1,4 +1,114 @@
+//Project 2: Beat That (Base+Comfortable)
+
+var rollDice = "ROLL DICE";
+var chooseSequence = "CHOOSE SEQUENCE";
+var gameMode = rollDice;
+var player1;
+var player2;
+var currentPlayer = 1;
+var player1Result = []; //player1'sDiceValues
+var player2Result = []; //player2'sDiceValues
+
+var diceRoll = function () {
+  return Math.floor(Math.random() * 6 + 1);
```

Comment:
Good job reducing this into one line! `Math.ceil(Math.random() * 6)` also works the same (ceil rounds up which is the same as floor + 1)